tasks/release.py

```diff
@@ -387,6 +393,31 @@ def list_major_change(_, milestone):
         print("#{}: {} ({})".format(pr["number"], pr["title"], pr["html_url"]))
 
 
+#
+# release.json manipulation invoke tasks section
+#
+
+##
+## I/O functions
+##
+
+
+def _load_release_json():
+    with open("release.json", "r") as release_json_stream:
+        return json.load(release_json_stream, object_pairs_hook=OrderedDict)
+
+
+def _save_release_json(release_json):
+    with open("release.json", "w") as release_json_stream:
+        # Note, no space after the comma
+        json.dump(release_json, release_json_stream, indent=4, sort_keys=False, separators=(',', ': '))
+
+
+##
+## Utils
+##
+
+
 def _is_version_higher(version_1, version_2):
```

Comment:
This would mean a bit more work but: have you considered having a `version` class? That way we can have
- an `__lt__` method to check if a version is higher than another
- a `__str__` method and we mostly won't have to use `_stringify_version` explicitly
- an `is_rc` method to check if something is an RC and
- a `next_version` method to get the next version.

I think that would clean up the code a bit

Reply:
I added a `Version` class in `libs` to deal with versions.